src/main/java/seedu/address/logic/parser/FindByCommandParser.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.FindByCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.FieldContainsKeywordsPredicate;
+
+/**
+ * Parses input arguments and creates a new FindByCommand object
+ */
+public class FindByCommandParser implements Parser<FindByCommand> {
+
+    private static final String[] fieldTypes = {"name", "email", "phone", "matric", "tag", "group"};
```

Comment:
Should we change this to e/, p/, t/ etc. for consistency?

Reply:
As discussed in class, let's not require fieldTypes.
But, maybe we can make it optional. For example, if you only want to search for "ab" in emails and not names of persons.